modules/ossm-api-settings-mesh-ha-autoscaling.adoc

```diff
@@ -0,0 +1,30 @@
+// This module is used in the following assemblies:
+// * service-mesh-docs-main/install/ossm-installing-service-mesh.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="ossm-api-settings-mesh-ha-autoscaling_{context}"]
+= API settings for Service Mesh HA autoscaling mode
+
+Use the following `istio` custom resource definition (CRD) parameters when you configure a service mesh for High Availability (HA) by using autoscaling. 
+
+.HA API parameters
+[cols="1,1"]
+|===
+|Parameter |Description 
+
+|`autoScaleMin` | Defines the minimum number of `istiod` pods for an istio deployment. Each pod contains one instance of the Istio control plane. 
+
+{ocp-short-name} only uses this parameter when the {istio} deployment uses the Horizontal Pod Autoscaler (HPA) configuration. 
```

Comment:
This note is a bit confusing. Creation of HPA is controlled by `Values.autoscaleEnabled`

Reply:
done